core/modules/css/PartTableParams.h

```diff
@@ -100,6 +104,16 @@ struct PartTableParams {
         if (not dirColName.empty()) res.push_back(dirColName);
         return res;
     }
+
+    /// Returns true if table should be locked in memory for shared scan
+    bool lockInMemory() const {
+        return lockInMem;
+    }
+
+    /// Returns the prefered speed of shared scan for the table
+    int getScanSpeed() const {
```

Comment:
I only added it because we had similarly trivial accessors there. Ok, I removed them.